content/national-qualifications-framework/ncit/content/test-driven-development/_index.md

```diff
@@ -49,8 +49,7 @@ ncit_standards:
 - 115359
 - 115384
 prerequisites:
-  hard:
-  - language-agnostic/validate-id-number
```

Comment:
are you sure?

Reply:
Yeah, I'm sure, basically moving these as they project should not block the topic. but the project should block the assessment